tests/test_variogram_unstructured.py

```diff
@@ -217,6 +217,101 @@ def test_assertions(self):
             ValueError, vario_estimate_unstructured, [x], field_e, bins
         )
 
+    def test_angles(self):
+        x = np.array([
+        -4.86210059, -4.1984934 , -3.9246953 , -3.28490663, -2.16332379,
+        -1.87553275, -1.74125124, -1.27224687, -1.20931578, -0.2413368 ,
+         0.03200921,  1.17099773,  1.53863105,  1.64478688,  2.75252136,
+         3.3556915 ,  3.89828775,  4.21485964,  4.5364357 ,  4.79236969]),
+        field = np.array([
+        -1.10318365, -0.53566629, -0.41789049, -1.06167529,  0.38449961,
+        -0.36550477, -0.98905552, -0.19352766,  0.16264266,  0.26920833,
+         0.05379665,  0.71275006,  0.36651935,  0.17366865,  1.20022343,
+         0.79385446,  0.69456069,  1.0733393 ,  0.71191592,  0.71969766])
+        
+        gamma_exp = np.array([
+        0.14260989, 0.18301197, 0.25855841, 0.29990083, 0.67914526,
+        0.60136535, 0.92875492, 1.46910435, 1.10165104])
+        
```

Comment:
Could you generate these values in a more "automated" way, as you described in the PR?

Reply:
I am happy to generate them automatically, however I learned to implement unit test with as little integration as possible with other components, in order to test functionality isolated, and not have the test fail because the test case generation failed. 

But thats a matter of philosophy in software development I guess. I will generate the automatically as you requested.

Reply:
for my own records: DONE

Reply:
Hmm, that's a good point I never thought about. Thanks for the input.

In the end, this is your work. I'll let you decide how to set up the tests.